### ISSUE TITLE
feat: verify accessibility for the accordion block

### ISSUE THREAD

**Issue by ramboz** [hlxsites/petplace#85]
Since the accordion block is already leveraging the `aria-accordion.js`, we should have pretty strong compliance to begin with.

Still, we can run this through:
- https://wave.webaim.org/ or https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh
- https://accessibe.com/accessscan
- https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd
